client/package.json

```diff
@@ -74,8 +74,10 @@
 		"@sentry/react": "^7.33.0",
 		"@stripe/react-stripe-js": "^1.4.1",
 		"@stripe/stripe-js": "1.17.1",
+		"@tanstack/query-sync-storage-persister": "^4.26.1",
```

Comment:
We use `query-sync-storage-persister` only in tests, so to be 100% correctness the package should be a `devDependency`.